website/docs/install/install-with-bazel.md

```diff
@@ -47,13 +47,25 @@ These hardware specifications are recommended, but not required to run the Prysm
 
 Instead of using the `Go` tool to build Prysm, our team relies on the [Bazel](https://bazel.build) build system used by major companies to manage monorepositories. Bazel provides reproducible builds and a sandboxed environment that ensures everyone building Prysm has the same experience and can build our entire project from a single command. For more detailed rationale on why Bazel, how it works in Prysm, and all important information about how exactly building from source works, read our rationale [here](/docs/reading/bazel).
 
+## Install Bazel using Bazelisk
+
+Bazelisk is a launcher for Bazel which automatically downloads and installs an appropriate version of Bazel. Use Bazelisk to automtically manage the version of Bazel required.  
+
+You can install Bazelisk in multiple ways, including:
+
+* npm install -g @bazel/bazelisk
+* Using a binary release for Linux, macOS, or Windows [(Download binaries)](https://github.com/bazelbuild/bazelisk/releases)
+* Using Homebrew on macOS
+* By compiling from source using Go: go get github.com/bazelbuild/bazelisk
+
 ## Building Prysm from source
 
-1. Open a terminal window. Ensure you are running the most recent version of Bazel by issuing the command:
+1. Open a terminal window. Ensure you are running ***Bazel*** through the ***Bazelisk*** launcher by issuing the command: 
```

Comment:
For ease of use, we should recommend to users to install bazelisk as bazel or alias bazelisk as bazel. Bazelisk looks and feels just like bazel, except that it will manage the correct bazel version for repositories with a `.bazelversion` file. 

Can we add a comment to recommend the following?

> Install it as the bazel binary in your PATH (e.g. copy it to /usr/local/bin/bazel). Never worry about upgrading Bazel to the latest version again.

From: https://github.com/bazelbuild/bazelisk/blob/master/README.md#installation

Personally, I have this in my `.bashrc` file.
```
alias bazel=bazelisk
```